tests/Behat/Page/Admin/CreditMemoDetailsPage.php

```diff
@@ -126,9 +126,20 @@ public function getToAddress(): string
         return $this->getDocument()->find('css', '#to-address')->getText();
     }
 
+    public function isCreditMemoInPosition(string $creditMemo, int $position): bool
+    {
+        $result = $this->getElement('credit_memo_in_given_position', [
+            '%position%' => $position,
+            '%creditMemo%' => $creditMemo,
+        ]);
+
+        return $result !== null;
```

Comment:
I guess it should work:
```suggestion
        return $this->hasElement('credit_memo_in_given_position', [
            '%position%' => $position,
            '%creditMemo%' => $creditMemo,
        ]);
```